services/dbmaker/controller.go

```diff
@@ -31,10 +31,24 @@ func (s *server) Create(ctx context.Context, body *pb.RequestBody) (*pb.Response
 		return nil, err
 	}
 
+	filter := types.M{
+		"owner": body.GetOwner(),
+		"instance_type": "database",
+		"date": time.Now().Format("2006-02-01"),
```

Comment:
Maybe you can create a helper function for this and pass instance type to create a filter.